feature/eks-cluster-creation/vars.tf

```diff
@@ -1,12 +1,16 @@
-// -- [ VARS ] ------------------------------------------------------
-// Cluster name
-variable "eks_cluster_name" {
-  default = "eks-terraform-cluster"
-}
-
 // Cluster version
 variable "eks_cluster_version" {
-  default = "1.17"
+  // default = "1.17"
+  default = "1.16"
```

Comment:
why downgraded k8s version?

Reply:
This is just to show, that version 1.16 is supported too. I just changed it to 1.17